mediagrains_py36/numpy/__init__.py

```diff
@@ -218,6 +218,15 @@ def __repr__(self):
         else:
             return "{}({!r},< numpy data of length {} >)".format(self._factory, self.meta, len(self.data))
 
+    def convert(self, fmt: CogFrameFormat) -> "VIDEOGRAIN":
```

Comment:
Minor: I don't think the return result type annotation requires quotes as VIDEOGRAIN type is known.